apps/web/components/apps/SkeletonLoader.tsx

```diff
@@ -4,10 +4,10 @@ import { SkeletonText } from "@calcom/ui";
 
 import { ShellSubHeading } from "@components/Shell";
 
-function SkeletonLoader() {
+function SkeletonLoader({ className }: { className?: string }) {
   return (
     <>
-      <ShellSubHeading title={<div className="h-6 w-32 rounded-sm bg-gray-100"></div>} />
+      <ShellSubHeading title={<div className="h-6 w-32 rounded-sm bg-gray-100"></div>} {...{ className }} />
```

Comment:
Interesting way to add a property

Reply:
haha I was kind of lazy it seems, and instead of duplicating the name of the property and its value, it came to me to do this

Reply:
It's ok to do this, but let's not fall into common pitfalls when doing this. I've seen how ugly this rabbit hole can go when abused.